webextensions/chrome/background.js

```diff
@@ -301,8 +301,8 @@ const ThinBridgeTalkClient = {
     }
     console.log(`* Result: [${matchedSectionNames.join(', ')}]`);
 
-    if (redirectCount == 0) {
-      console.log(`* No redirection: fallback to default`);
+    if (redirectCount == 0 && loadCount == 0) {
```

Comment:
It might be better to add early return for `"load"` case, v1.4.0's code is like this

https://github.com/ThinBridge/ThinBridge/blob/973debb5d97d63d40b42ab98eb0b4d710ff04902/webextension/edge/background.js#L214-L252